docs/UserGuide.adoc

```diff
@@ -474,35 +493,53 @@ Examples:
+optionally attach a supplementary note to said paragraph. You can add notes
+to justify the highlight or as content-relevant notes to refer to in future.
 
-NOTE: A general note is known to attach to a <<phantom-paragraph,_phantom paragraph_>>. As you can see in the offline
-copy, phantom paragraphs do not have any content displayed; they do not belong to the original web page.
-The phantom paragraphs altogether make up the _General notes section_.
+If you are looking to add a general note not pertaining to any specific paragraph, you
+can also use this command to add it to the bottom of the page, hereby known as the _General notes section_.
+
+Upon annotating, the your view will be switched to the offline tab
+showing the results of your command.
+
+NOTE: A general note is not attached to any paragraph of the original web page. Instead, when adding general notes, they are attached
+to newly created paragraphs that do not have any content displayed.
+General notes are found at the bottom of the page, referred to as the _General notes section_.
 
 You can choose to use this command to overwrite any existing note. However, note that when you use this command to highlight
 a phantom paragraph, Mark remembers the hidden highlight colour, but no highlight is reflected on the application.
 
-
-NOTE: Paragraphs are identified using a numbered identifier that starts with either `P` or `S`. You can refer to the
+NOTE: Paragraphs are identified using a numbered identifier that starts with either `P` or `G`. You can refer to the
 numbered identifier in the leftmost column of the offline document to check out the respective identifier for each paragraph.
-Phantom paragraphs are identified using numbered identifiers that begin with `S`, as opposed to <<true-paragraph,_true paragraphs_>>, whose identifiers begin with `P`.
+Paragraphs of general notes are identified using numbered identifiers that begin with `G`, as opposed to paragraphs originating from the original web page, whose identifiers begin with `P`.
 
 Format: `*annotate* INDEX p/PARA_NUM [n/NOTES] [h/HIGHLIGHT_COLOUR=yellow]`
 
+For example:
+
+* Input `*annotate* 1 p/p2 n/summary of paragraph h/orange` into the command box.
+
+image::AddAnnotationCommandUi1.png[]
+// offline tab, showing result after   annotate 1 p/p2
+
+* Observe that any pre-existing annotation is overwritten. Paragraph P2 is now highlighted orange and a note with content “summary of paragraph” is attached to it.
+
+image::AddAnnotationCommandUi2.png[]
+// offline tab, showing result.
+
+
+// More details about command constraints
 ****
  * `INDEX` is the bookmark that you want to annotate offline version of.
  If `INDEX` is invalid, a warning message will be displayed.
  * `PARA_NUM` is the numbered identifier of the paragraph to be marked.
- `PARA_NUM` must be `NULL` or it must begin with `P` or `S` (e.g. `P3`).
+ `PARA_NUM` must be `NULL` or it must begin with `P` or `G` (e.g. `P3`).
```

Comment:
Just a suggestion should we mention that `p3` and `P3` are equivalent i.e. para_num is case insensitive?

Reply:
Ok!